python/paddle/optimizer/optimizer.py

```diff
@@ -91,6 +95,29 @@ class Optimizer(object):
             adam.step()
             adam.clear_grad()
 
+            #Take the subclass sgd as an example
+            #optimize parameters in linear_1 and linear2 in different options. 
+            #Note that the learning_rate of linear_2 is 0.01.
```

Comment:
the learning_rate of linear_2 is 0.01 ?

Reply:
now the learning_rate in param groups is the scale of base learning rate, thus the lr of linear_2 is 0.1*0.1 = 0.01